packages/utils/src/hl7v2-notifications/hl7-to-fhir-converter-logic.ts

```diff
@@ -51,22 +48,22 @@ const filePath = "";
 const fileName = "";
 
 function invokeLambdaLogic() {
+  const hl7WebhookLambda = buildHl7NotificationWebhookSender();
```

Comment:
the pattern is usually to build this where it is being invoked. Recomputing is not an issue, the constructor is not doing anything of substance.